BlackJackets-ui/src/App.jsx

```diff
@@ -55,6 +62,9 @@ function App() {
                   path="gigs/add"
                   element={<CreateGigForm venueId={venueId} />}
                 />
+             <Route path="/band-message" element={<MessageAdd />} />
```

Comment:
The two routes on lines 65 and 66 should be moved up above the conditional statement on line 59 so that they can be accessed by a guest.